### ISSUE TITLE
Simulation crashes when switching JawRotate as revolute joint

### ISSUE THREAD

**Issue by torydebra** [freefloating-gazebo/freefloating_gazebo_demo#4]
I'm trying to use FreeFloating following the demo. 
When I switch JawRotate joint from fixed to revolute, gazebo sends the robot outside the visible region. 
Specifically in the file urdf/arm5e.xacro, line 137:
```xml
<!--<joint name="JawRotate" type="revolute">
    <parent link="part3"/>
    <child link="part4_base"/>
    <axis xyz="1 0 0"/>
    <origin rpy="0 0 0" xyz="0.320 0 0.077 "/>
    <dynamics damping="0.8"/>
    <limit lower="${-PI/2}" upper="${PI/2}" velocity="1" effort="200"/>
    </joint> -->
    
    <joint name="JawRotate" type="fixed">
    <parent link="part3"/>
    <child link="part4_base"/>
    <axis xyz="1 0 0"/>
    <origin rpy="0 0 0" xyz="0.320 0 0.077 "/>
    </joint>
```
I decommented the first definition of JawRotate and commented the second, then launched the simulation:
`roslaunch freefloating_gazebo_demo g500arm5e.launch parse:=true`
`roslaunch freefloating_gazebo_demo g500arm5e.launch` / `roslaunch freefloating_gazebo_demo g500arm5e_gazebo.launch`   (both cause same issue)
`rosservice call /gazebo/unpause_physics`
when I unpause the physics, robot goes away.

**Comment by torydebra** [comment]
It seems a problem of inertia matrices. I change jaw_mass from 0.1 to 0.5 in :
```xml
  <xacro:property name="jaw_mass" value="0.1"/>
```
According to ros doc:
> When using realtime controllers, inertia elements of zero (or almost zero) can cause the robot model to collapse without warning, and all links will appear with their origins coinciding with the world origin.

In fact everything collapsed in the origin.
Altought I dont know how much is "almost zero", problems were given by values of 0.0001 for the inertia matrix.  (with 0.5 mass inertia becomes 0.0005)

**Comment by oKermorgant** [comment]
Hi, 

Thanks for this issue. Indeed I had deactivated this joint in order to show a simpler demo.
The original URDF comes from UWSim developpers and they were using their own dynamic simulation.

As a consequence, the values did not seem to be suited for Gazebo. It was ignored when the joint was fixed as in this case Gazebo fuses all the links into a single one, building its corresponding mass and inertia matrix.

I have committed the changes and the corresponding control interface demo.
